README.md

```diff
@@ -24,9 +24,22 @@ Default Docker image is [yegor256/rultor](https://registry.hub.docker.com/u/yego
 
 TBD... _product statement_
 
-## What Problem Does Rultor Solve?
+## What Problems Does Rultor Solve?
 
-TBD... _stakeholders and needs_
+Automated build scripts have been around for some time. Rultor attempts to
```

Comment:
[Yegor compares them](http://www.yegor256.com/2014/09/11/deployment-script-vs-rultor.html) and I think it is fair. Although underlying there is an invocation, Rultor does more than building the application as it also manages software releases.

Maybe deployment scripts would better describe what I wanted.